09_deploy_kubevirt.sh

```diff
@@ -36,3 +37,4 @@ oc apply -f operator.yaml
 sed "s/okdvirt/openshiftvirt/" crds/kubevirt_v1alpha1_kwebui_cr.yaml > crds/kubevirt_v1alpha1_kwebui_cr-modified.yaml
 sed -i "s/version: .*/version: \"$UIVERSION\"/" crds/kubevirt_v1alpha1_kwebui_cr-modified.yaml
 oc apply -f crds/kubevirt_v1alpha1_kwebui_cr-modified.yaml
+oc project ${CURRENT_PROJECT}
```

Comment:
If anything fails in the script we'll fail to revert the project, I think we should run this via a trap, e.g something like:

  trap "oc project ${CURRENT_PROJECT}" EXIT

Set near the start of the script, e.g immediately after we read the current project, or immediately after we change it?

Reply:
> If anything fails in the script we'll fail to revert the project, I think we should run this via a trap, e.g something like:
> 
> `trap "oc project ${CURRENT_PROJECT}" EXIT`
> 
> Set near the start of the script, e.g immediately after we read the current project, or immediately after we change it?
Yes, it would be safer this way.